src/main/twirl/com/lightbend/lagom/docs/main.scala.html

```diff
@@ -15,6 +15,22 @@
         <meta name="viewport" content="width=device-width,maximum-scale=1"/>
         <link href="//fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700" rel="stylesheet" type="text/css"/>
         <link rel="stylesheet" href="@context.path/@context.assetFingerPrint-all-styles-concat.css" type="text/css">
+        <!-- OneTrust Cookies Consent Notice (Production Standard, lagomframework.com, en-GB) start -->
+        <script src="https://optanon.blob.core.windows.net/consent/1ee3f0d6-9ed5-4b9e-bed7-ce6faaeaca5d.js" type="text/javascript" charset="UTF-8"></script>
```

Comment:
@ignasi35 When testing locally you'll need to swap this include out for the testing version: `<script src="https://optanon.blob.core.windows.net/consent/1ee3f0d6-9ed5-4b9e-bed7-ce6faaeaca5d-test.js" type="text/javascript" charset="UTF-8"></script>`